src/types/mod.rs

```diff
@@ -168,6 +168,11 @@ impl Flag<'static> {
             _ => None,
         }
     }
+
+    /// Helper function to transform a [`Vec`] of flag strings into a [`Vec`] of [`Flag`].
+    pub fn from_vec<S: ToString>(v: &[S]) -> Vec<Self> {
```

Comment:
Let's not enforce an allocation here (e.g., for passing this to `extend`). For maximal re-use, try:
```rust
impl Flag<'a> {
    pub fn from_strs<S>(v: impl IntoIterator<Item = S>) -> impl Iterator<Item = Self> where Flag<'a>: From<S> {
      v.into_iter().map(|s| Flag::from(s))
    }
}
```

Reply:
Ah, nice. Will do. :)